### ISSUE TITLE
a bug when yValue is infinite

### ISSUE THREAD

**Issue by LeBW** [petervanhoef/Graphing-Calculator#21]
For example, when I draw tan(M), there is a straight vertical line at (x = π/2).  Have you noticed that?

**Comment by petervanhoef** [comment]
Yes, already during coding I realised that this would go wrong. The current logic doesn't detect discontinuities of a function and just connects all the calculated y-values with each other.
